app/views/shared/_navigation.html.haml

```diff
@@ -28,6 +28,4 @@
           = link_to t("page_title.users.index"), users_path, class: "govuk-header__link"
 
       %li{ class: navigation_item_class(sign_out_path) }
-        %a{ href: sign_out_path, class: 'govuk-header__link' }
-          = t("header.link.sign_out")
-
+        = link_to t("header.link.sign_out"), destroy_user_session_path, method: :delete, class: "govuk-header__link"
```

Comment:
I'm suspicious of `link_to ... method: :delete` - I may be out of date here, but an `a` tag should be an immutable get request, and I think it can cause unexpected issues if this assumption isn't kept.

Reply:
I think from a screen reader semantics point of view, yes. Originally @CristinaRO and I had changed this to a button, which then left us with a styling problem!